app/frontend/components/info_popup.js

```diff
@@ -121,6 +121,17 @@ class InfoPopup {
       `;
     }
 
+    function urlify(text) {
+      // replace urls in text with <a> html tag
+      // regex extracted from https://www.codegrepper.com/code-examples/whatever/use+regex+to+get+urls+from+string
+      var urlRegex = /(?:(?:https?|ftp|file):\/\/|www\.|ftp\.)(?:\([-A-Z0-9+&@#\/%=~_|$?!:,.]*\)|[-A-Z0-9+&@#\/%=~_|$?!:,.])*(?:\([-A-Z0-9+&@#\/%=~_|$?!:,.]*\)|[A-Z0-9+&@#\/%=~_|$])/igm
+      return text.replace(urlRegex, function(url) {
+        console.log(url)
+        return '<a href="' + url + '" target="_blank" title="' + url + '">' + url + '</a>';
```

Comment:
2 things

* Remove the console.log - this is only for debugging
* I would suggest to use a [template literal](https://www.w3schools.com/js/js_string_templates.asp) for putting the variables into the markup.

Reply:
- Have removed the console.log in the latest changes, ty for spotting that.
- The title attribute enables the URL to appear on hover over the link. I have removed it in latest changes. 
- Change made.

I wasn't sure the best place to put the "urlify" function, any suggestions.